Makefile

```diff
@@ -1,58 +1,61 @@
 PIP_HOME = $(shell python3 -c "import site; import os; print(os.path.join(site.USER_BASE, 'bin'))" \
 	|| python -c "import site; import os; print(os.path.join(site.USER_BASE, 'bin'))")
 
+BIN := .venv/bin
+PYTHON := $(BIN)/python
+
 IP ?= 127.0.0.1
 PORT ?= 3000
 
 .PHONY: dev
 dev:
-	HKNWEB_MODE='dev' pipenv run python ./manage.py runserver $(IP):$(PORT)
+	HKNWEB_MODE='dev' $(PYTHON) ./manage.py runserver $(IP):$(PORT)
 
 .PHONY: livereload
 livereload:
-	HKNWEB_MODE='dev' pipenv run python ./manage.py livereload $(IP):$(PORT)
-
-setup: pipenv venv migrate
```

Comment:
I notice you've lost some make dependencies here, it used to do more setup (make a virtualenv, install packages, run migrations), where now it just created the virtualenv but doesn't install any packages). Maybe this should be a target called `venv` instead and `setup` could be the same as before?

Reply:
That's a good idea -- the current flow is `make setup` --> `source .venv/bin/activate` --> `make install`, but `make venv` makes a lot more sense